skunkworks_crow/src/main/res/layout/toolbar.xml

```diff
@@ -7,7 +7,7 @@
     android:background="?attr/colorPrimary"
     android:minHeight="?attr/actionBarSize"
     app:contentInsetEnd="0dp"
-    app:contentInsetStart="0dp"
+    app:contentInsetStart="16dp"
```

Comment:
Please undo this change as it is not related

Reply:
@shobhitagarwal1612 Sir, Undid it.